<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_Students</class>
 <widget class="QDialog" name="edit_Students">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Full Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>E-mail :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Password :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Phone : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Address :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="Line_name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_email">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_password">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_phone">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_address">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Year :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="y1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="y2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="y3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="y4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="y5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Group :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Combo_Student">
     <item>
      <property name="text">
       <string>Group 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group 12</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Date">
     <property name="text">
      <string>Date Of Birth :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="Day_3">
       <property name="text">
        <string>Day :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="L_Day_3"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Month_3">
       <property name="text">
        <string>Month :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QLineEdit" name="L_Month_3"/>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QLabel" name="Year_3">
       <property name="text">
        <string>Year :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="L_Year_3"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>347</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancelStudent">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditStudent">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
